src/views/AddItem.jsx

```diff
@@ -1,3 +1,21 @@
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	return (
```

Comment:
@xynree I love love LOVE seeing an outline for the layout of the form, before any hooks are added in, this is such a great habit to have. Early and often commits are 🔥  great work!